public/docs/_examples/user-input/dart/lib/todo_list.dart

```diff
@@ -0,0 +1,41 @@
+library user_input.todo_list;
+
+import 'dart:html';
+
+import 'package:angular2/angular2.dart';
+
+@Component(selector: 'todo-list')
+@View(
```

Comment:
I'm using @View in the other samples (pipes and forms) that has already been merged. Should i update those and do a new PR ?